packages/checkout-ui-extensions/src/components/Choice/README.md

```diff
@@ -0,0 +1,13 @@
+# Choice
+
+Options inside a ChoiceList.
+The wrapping ChoiceList component will dictate if the Choice renders as radio buttons or checkboxes.
+
+## Props
+optional = ?
+
+| Name | Type | Description |
+| --- | --- | --- |
+| id | <code>string</code> | A unique identifier for the choice.  |
+| disabled? | <code>boolean</code> | Whether the choice can be changed.  |
+| accessibilityLabel? | <code>string</code> | A label to use for the choice that will be used for buyers using assistive technologies. When provided, any `children` supplied to this component are hidden from being seen for accessibility purposes to prevent duplicate content from being read.  |
```

Comment:
```suggestion
| accessibilityLabel? | <code>string</code> | A label to use for the choice that will be used for buyers using assistive technologies. When provided, any `children` supplied to this component are hidden from being seen for accessibility purposes to prevent duplicate content from being read.  |
```

>A label to use for the choice that will be used for buyers using 

This is difficult to understand, can you revise it to be clearer?

Reply:
yeah I agree. I'll rewrite it in another PR (in checkout-web)